src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -88,10 +88,10 @@ export class CellModel extends Disposable implements ICellModel {
 
 	constructor(cellData: nb.ICellContents,
 		private _options: ICellModelOptions,
+		@IAdsTelemetryService private _telemetryService?: IAdsTelemetryService,
```

Comment:
Just undo this change - the services shouldn't be optional here but no reason to change this now that I can see